docs/csharp/programming-guide/classes-and-structs/snippets/local-functions/Program.cs

```diff
@@ -121,5 +121,27 @@ private async Task<int> SecondStep(int index, string name)
             await Task.Delay(100);
             return 9;
         }
+
+        //<YieldReturn>
+        public IEnumerable<string> SequenceToLowercase(IEnumerable<string> input)
```

Comment:
The project doesn't build now because of the missing `using` directive. Please add `using System.Collections.Generic;` to the top of the file.

Reply:
Ah! I'm embarrassed 😞 - I'll fix this first

I'm using Visual Studio Code to make my changes; is there a way to run a local build to ensure I'm not forgetting anything that would break the actual build in the future?

Reply:
Since this code is inline, there isn't. 😢  

We've been working toward using all code includes, but we haven't completed that transition yet. Where those are in place, all the displayed code runs through our CI system.

Reply:
@jmarkman just build the project in the corresponding folder: `docs/csharp/programming-guide/classes-and-structs/snippets/local-functions`